src/frontend/components/app/language-select/language-select.tsx

```diff
@@ -12,7 +12,11 @@ const LanguageButtons: FC = () => {
   const handleButton = (lng: string) => {
     const selectedLocale = { ...locales[lng] }
     selectedLocale.availableLanguages = availableLanguages
-    setLocale(selectedLocale)
+    if (locale.language === lng) {
+      setLocale(locale)
```

Comment:
Do we have to change the language if locale is the same? Or is this some other logic?

Reply:
fixed